db/seeds.rb

```diff
@@ -8,6 +8,7 @@
 
 require 'open-uri'
 
+Review.destroy_all
 Booking.destroy_all
 Galaxy.destroy_all
 User.destroy_all
```

Comment:
do we need to destroy it every time? cloudinary can be overwhelmed during reseed testing.
also i dont mind running seed couple of times in a row.

i believe delete methods can be replaced with db:seed:repland